HangmanApp.js

```diff
@@ -0,0 +1,141 @@
+document.addEventListener('DOMContentLoaded',() =>{
```

Comment:
Overall good project. One caveat is that when the player uses same input that has already been used, the game still adds that letter to letters guessed. It would be useful for game to show some text or window alert to let player know that when a word has been used already. Nice background and font for the main screen. Try to fix that aforementioned glitch when you get a chance. Keep up the good work!